Makefile

```diff
@@ -71,4 +47,50 @@ endif
 ifndef HAS_GOLANGCI
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin
 endif
-	dep ensure
\ No newline at end of file
+	dep ensure
+
+################################################################################
+# Tests                                                                        #
+################################################################################
+
+.PHONY: lint
+lint:
+	golangci-lint run --config ./golangci.yml
+
+.PHONY: test
+test:
+	go test ./pkg/...
+
+################################################################################
+# Build / Publish                                                              #
+################################################################################
+
+BINS := github-gateway check-run
+IMAGES = brigade-github-app brigade-github-check-run
+
+# Cross-compile for Docker+Linux
+.PHONY: build-all-bins
```

Comment:
We'll need to update the corresponding (though admittedly sparse) docs in the README, e.g. at https://github.com/brigadecore/brigade-github-app/blob/master/README.md#building-from-source

(Exercise left to the author on whether or not to just move that section to the pre-existing `DEVELOPERS.md` doc and link from the README...)

Reply:
Thanks for keeping me honest on doc changes-- I too often forget those.

Refer back to [this comment](https://github.com/brigadecore/brigade-github-app/pull/59#discussion_r287177202)

My strong preference (if our cluster supports it) is to stop building binaries and images as two separate steps (not doing so atomically is just one extra thing that can go wrong in the process)...

And yes... I'd update the docs accordingly. 😄